docs/typescript/tutorials/instantiated-objects.md

```diff
@@ -4,96 +4,57 @@ title: Using Instantiated Objects
 description: Learn how to make workflows, transactions, and steps reusable and configurable by instantiating objects
```

Comment:
Let's rename this guide something like "Working with TypeScript Classes", echoing Python?  No one will know what "instantiated objects" are.